front-end/src/__tests__/Sidebar.test.js

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
```

Comment:
I ran the tests with this line removed and it worked fine. Please take it out so we don't forget to do that in the future.